src/views/AboutView.vue

```diff
@@ -22,6 +22,11 @@
     </div>
     <div class="control-group">
       <div class="control">
+        <a
+          class="btn"
+          href="https://github.com/xenharmonic-devs/scale-workshop/wiki"
```

Comment:
I would suggest adding `target="_blank"` to open the guide in a new tab. After that, looks good to merge in